.aspect/workflows/terraform/workflows.tf

```diff
@@ -0,0 +1,76 @@
+data "google_compute_image" "runner_image" {
+  # Aspect's GCP aspect-workflows-images project provides public Aspect Workflows GCP images for
+  # getting started during the trial period. We recommend that all Workflows users build their own
+  # GCP images and keep up-to date with patches. See
+  # https://docs.aspect.build/v/workflows/install/packer for more info and/or
+  # https://github.com/aspect-build/workflows-images for example packer scripts and BUILD targets
+  # for building GCP images for Workflows.
+  project = "aspect-workflows-images"
+  name    = "aspect-workflows-ubuntu-2304-docker-gcc-make-1-1-0"
+}
+
+module "aspect_workflows" {
+  # Aspect Workflows terraform module
+  source = "gcs::https://storage.googleapis.com/storage/v1/aspect-artifacts/5.7.0-rc8/workflows-gcp/terraform-gcp-aspect-workflows.zip"
```

Comment:
rc9?

Reply:
it will end up being rc10 since I have to cut one more and that will include a scale-in fix for GCP + Buildkite